docs/source/en/api/pipelines/wuerstchen.md

```diff
@@ -0,0 +1,110 @@
+- Faster inference
+- Multi Aspect Resolution Sampling
+- Better quality
+
+## Text-to-Image Generation
+
+For the sake of usability Würstchen can be used with a single pipeline. This pipeline is called `WuerstchenPipeline` and can be used as follows:
+
+```python
+import torch
+from diffusers import WuerstchenPipeline
+
+device = "cuda"
+dtype = torch.float16
+num_images_per_prompt = 2
+
+pipeline = WuerstchenPipeline.from_pretrained(
+    "warp-diffusion/WuerstchenPipeline", torch_dtype=dtype
+).to(device)
+
+caption = "A captivating artwork of a mysterious stone golem"
+negative_prompt = "bad anatomy, blurry, fuzzy, extra arms, extra fingers, poorly drawn hands, disfigured, tiling, deformed, mutated"
+
+output = pipeline(
+    prompt=caption,
+    height=1024,
+    width=1024,
+    negative_prompt=negative_prompt,
+    guidance_scale=8.0,
+    num_images_per_prompt=num_images_per_prompt,
+    output_type="pil",
+).images
+```
+
+For explanation purposes, we can also initialize the two main pipelines of Würstchen individually. Würstchen consists of 3 stages: Stage C, Stage B, Stage A. They all have different jobs and work only together. When generating text-conditional images, Stage C will first generate the latents in a very compressed latent space. This is what happens in the `prior_pipeline`. Afterwards, the generated latents will be passed to Stage B, which decompresses the latents into a bigger latent space of a VQGAN. These latents can then be decoded by Stage A, which is a VQGAN, into the pixel-space. Stage B & Stage A are both encapsulated in the `decoder_pipeline`. For more details, take a look the [paper](https://huggingface.co/papers/2306.00637).
+
+```python
+import torch
+from diffusers import WuerstchenDecoderPipeline, WuerstchenPriorPipeline
+
+device = "cuda"
+dtype = torch.float16
+num_images_per_prompt = 2
+
+prior_pipeline = WuerstchenPriorPipeline.from_pretrained(
+    "warp-diffusion/WuerstchenPriorPipeline", torch_dtype=dtype
+).to(device)
+decoder_pipeline = WuerstchenDecoderPipeline.from_pretrained(
+    "warp-diffusion/WuerstchenDecoderPipeline", torch_dtype=dtype
+).to(device)
+
+caption = "A captivating artwork of a mysterious stone golem"
+negative_prompt = "bad anatomy, blurry, fuzzy, extra arms, extra fingers, poorly drawn hands, disfigured, tiling, deformed, mutated"
+
+prior_output = prior_pipeline(
+    prompt=caption,
+    height=1024,
+    width=1024,
+    negative_prompt=negative_prompt,
+    guidance_scale=8.0,
+    num_images_per_prompt=num_images_per_prompt,
+)
+decoder_output = decoder_pipeline(
+    predicted_image_embeddings=prior_output.image_embeds,
+    prompt=caption,
+    negative_prompt=negative_prompt,
+    num_images_per_prompt=num_images_per_prompt,
+    guidance_scale=0.0,
+    output_type="pil",
+).images
+
+```
+
+The original codebase, as well as experimental ideas, can be found at [dome272/Wuerstchen](https://github.com/dome272/Wuerstchen).
+
+## WuerschenPipeline
```

Comment:
@patrickvonplaten, does it make sense to allow WuerschenPipeline from the AutoPipeline API?

Reply:
Disregard. Already done here: https://github.com/huggingface/diffusers/pull/3849/files#diff-edd3c8f30ab707862237a901e184533c4d44c6bf26bb307e965ef9dda28444f0R67.